src/lib.rs

```diff
@@ -2533,6 +2533,21 @@ impl Build {
                 }
                 None => default_ar,
             }
+        } else if target.starts_with("wasm32") {
```

Comment:
Could this be `wasm` instead of `wasm32` to cover `wasm64` as well?

Reply:
Well, yes. But first let's consider rationale behind this specific choice. It was that `rustup target list` doesn't offer any wasm64 options. Nor is `wasm` used by itself anywhere in cc-rs, always as `wasm32`.